packages/playground/blueprints/src/lib/steps/run-sql.ts

```diff
@@ -0,0 +1,90 @@
+import { StepHandler } from '.';
+import { rm } from './rm';
+
+/**
+ * @inheritDoc runSql
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ *		"step": "runSql",
+ *		"sql": {
+ *			"resource": "literal",
+ *			"name": "schema.sql",
+ *			"contents": "SELECT * FROM wp_posts"
+ *		},
+ * }
+ * </code>
+ */
+export interface RunSqlStep<ResourceType> {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSql';
+	/**
+	 * The SQL to run.
+	 */
+	sql: ResourceType;
+	/**
+	 * Whether to stop on errors
+	 */
+	stopOnErrors?: boolean;
+}
+
+/**
+ * Run an SQL file.
+ */
+export const runSql: StepHandler<RunSqlStep<File>> = async (
+	playground,
+	{ sql, stopOnErrors = true },
+	progress?
+) => {
+	progress?.tracker.setCaption(`Executing SQL File`);
```

Comment:
Nitpick, queries may come from any resource, not just from files:

```suggestion
	progress?.tracker.setCaption(`Executing SQL queries`);
```

Reply:
Done: https://github.com/WordPress/wordpress-playground/pull/745/files#diff-ef7fc55c9e2c228ab627cd90d8372d926a0d8625f5f8fc76baf10cd15ed4f85fR40